schema/mysql/upgrades/1.1.2.sql

```diff
@@ -1 +1,10 @@
+ALTER TABLE host_state MODIFY COLUMN check_attempt int unsigned NOT NULL;
+
+ALTER TABLE service_state MODIFY COLUMN check_attempt int unsigned NOT NULL;
+
+ALTER TABLE state_history MODIFY COLUMN check_attempt tinyint unsigned NOT NULL COMMENT 'optional schema upgrade not applied yet, see https://icinga.com/docs/icinga-db/latest/doc/04-Upgrading/#upgrading-to-icinga-db-v112';
```

Comment:
Indeed, seems impossible to change the comment without repeating the column type: https://bugs.mysql.com/bug.php?id=64439

Thus, if it was applied in the wrong order, it would actually undo the type change. So the upgrading docs should explicitly state to apply it after `1.1.2.sql`.